metadata/network.go

```diff
@@ -23,30 +23,33 @@ import (
 
 // NetworkDefinition structure holds all parameters which describe particular network
 type NetworkDefinition struct {
-	DiscoveryAPIAddress     string
-	BrokerAddress           string
-	EtherClientRPC          string
-	QualityOracle           string
-	PaymentsContractAddress common.Address
+	DiscoveryAPIAddress               string
+	AccessPolicyOracleEndpointAddress string
+	BrokerAddress                     string
+	EtherClientRPC                    string
+	QualityOracle                     string
+	PaymentsContractAddress           common.Address
 }
 
 // TestnetDefinition defines parameters for test network (currently default network)
 var TestnetDefinition = NetworkDefinition{
-	"https://testnet-api.mysterium.network/v1",
-	"nats://testnet-broker.mysterium.network",
-	"https://ropsten.infura.io",
-	"https://testnet-morqa.mysterium.network/api/v1",
-	common.HexToAddress("0xbe5F9CCea12Df756bF4a5Baf4c29A10c3ee7C83B"),
+	DiscoveryAPIAddress:               "https://testnet-api.mysterium.network/v1",
+	AccessPolicyOracleEndpointAddress: "https://testnet-trust.mysterium.network/api/v1/access-policies/",
```

Comment:
We don't have a `Endpoint` in other addresses.
```suggestion
	AccessPolicyOracleAddress: "https://testnet-trust.mysterium.network/api/v1/access-policies/",
```